### DISCUSSION TITLE
Add Spoofing warning

### DISCUSSION THREAD

**Discussion by mrx23dot** [simple-login/app#1845] [Feature Requests]
By using simple-login I think I lose the ability to detect these:

![gmail-warning-message-632218685](https://github.com/simple-login/app/assets/4163396/1133ce47-0474-4d59-87e0-2f8bacd620d7)

because source address is replaced,

so it would be simple-login's job to put these back in.
I think they are based on SPF, DKIM and DMARC.

**Comment by LavaBit5** [comment]
I think it would be nice if SimpleLogin will automatically put such mails to SPAM folder, non only “mark” them

**Reply by mrx23dot** [reply]
Then you wouldn't know you are under attack, I never login into SimpleLogin.

**Comment by D1stract1on** [comment]
SL has something like this

![IMG_0163](https://github.com/simple-login/app/assets/144631117/93234a63-5a33-4105-8df9-2c9b9db16403)

But it will be nice if such mails will be automatically delivered into inbox or rejected at all (quarantined (SL stores quarantined emails for 7 days))

**Comment by mrx23dot** [comment]
seems to be already supported 

https://simplelogin.io/docs/getting-started/anti-phishing/

`If the email is flagged as spam/phishing, this means that the sender explicitly states their emails should respect DMARC (an email authentication protocol) and any email that violates this should either be quarantined or rejected. If possible, please contact the sender so they can update their DMARC setting or fix their SPF/DKIM that cause the DMARC failure. Their emails are probably being rejected or end up in spam at other email providers as well. `
